lib/absinthe/phase/schema/type_imports.ex

```diff
@@ -17,22 +17,35 @@ defmodule Absinthe.Phase.Schema.TypeImports do
     {Absinthe.Type.BuiltIns.Introspection, []}
   ]
   def handle_imports(%Schema.SchemaDefinition{} = schema) do
-    {types, schema} =
-      do_imports(@default_imports ++ schema.imports, schema.type_definitions, schema)
+    {types, type_extensions, schema} =
+      do_imports(
+        @default_imports ++ schema.imports,
+        schema.type_definitions,
+        schema.type_extensions,
+        schema
+      )
 
     # special casing the import of the built in directives
     [builtins] = Absinthe.Type.BuiltIns.Directives.__absinthe_blueprint__().schema_definitions
     directives = schema.directive_definitions ++ builtins.directive_definitions
-    {:halt, %{schema | type_definitions: types, directive_definitions: directives}}
+
+    schema = %{
+      schema
+      | directive_definitions: directives,
+        type_extensions: type_extensions,
```

Comment:
Do you still need this in this phase now that you added that other `ApplyTypeExtensions` phase?

Reply:
I've extracted the type extension imports to a separate phase, like with import_directives.